frontend/src/components/ui/DataTable.tsx

```diff
@@ -137,9 +137,9 @@ export function DataTable<TData, TValue>({
     }
   });
 
-  useKeySequence(["j"], handleNextRow);
-  useKeySequence(["k"], handlePrevRow);
-  useKeySequence(["Enter"], handleRowSelect);
+  useHotkeys(["j"], handleNextRow);
+  useHotkeys(["k"], handlePrevRow);
+  useHotkeys(["Enter"], handleRowSelect, { preventDefault: true });
```

Comment:
this prevents a bug with vite (i think) since hitting enter can also trigger a hot reload locally